src/Accelerators/NNPA/Conversion/ONNXToZHigh/NNPALimit.h

```diff
@@ -28,3 +28,9 @@ static constexpr int64_t NNPA_MAXIMUM_TENSOR_SIZE = 4294967296;
 // See zDNN API doc
 static constexpr int64_t MAXIMUM_NUM_HIDDEN_SIZE_LSTM = 8192;
 static constexpr int64_t MAXIMUM_NUM_HIDDEN_SIZE_GRU = 10880;
+
+// The NNPA levels.
+static constexpr const char *NNPA_Z13 = "z13";
+static constexpr const char *NNPA_Z14 = "z14";
+static constexpr const char *NNPA_Z15 = "z15";
```

Comment:
Do we use `NNPA_Z{13, 14, 15}` in practice?

Reply:
Thanks, I removed the definitions for NNPA_Z{13, 14, 15}